test/mix/tasks/appsignal_diagnose_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule Mix.Tasks.Appsignal.DiagnoseTest do
   alias Appsignal.{Diagnose.FakeReport, FakeNif, FakeSystem}
 
   @appsignal_version Mix.Project.config()[:version]
-  @agent_version Appsignal.Nif.agent_version()
+  @agent_version Appsignal.Agent.agent_version()
```

Comment:
Is this module file always loaded here, or do we need to manually load it like we do in the mix helpers? https://github.com/appsignal/appsignal-elixir/blob/c0a98928fb331436f01b7b41212decf7ab949ed1/mix_helpers.exs#L1

Reply:
As it is loaded by `mix.exs` through the mix helpers, we don't need to manually load it anywhere else within the lib.

Reply:
The mix helpers are only loaded in the compile step I think, it's not always present.

Reply:
We made some fixes in #745. I did some manual testing I locally and I couldn't get this to break. Probably because this is set during compilation and not called at runtime.